Shared/Samples/Apply dictionary renderer to graphics overlay/ApplyDictionaryRendererToGraphicsOverlayView.swift

```diff
@@ -0,0 +1,189 @@
+    @State private var scene = Scene(basemapStyle: .arcGISTopographic)
+    
+    /// The graphics overlay for displaying the message graphics on the scene.
+    @State private var graphicsOverlay = GraphicsOverlay()
+    
+    /// The camera for zooming the scene view to the message graphics.
+    @State private var camera: Camera?
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        SceneView(scene: scene, camera: $camera, graphicsOverlays: [graphicsOverlay])
+            .task {
+                do {
+                    // Sets up the graphics overlay when the sample opens.
+                    graphicsOverlay.renderer = try await makeMIL2525DRenderer()
+                    try graphicsOverlay.addGraphics(makeMessageGraphics())
+                    
+                    // Sets the camera to look at the graphics in the graphics overlay.
+                    guard let extent = graphicsOverlay.extent else { return }
+                    camera = Camera(
+                        lookingAt: extent.center,
+                        distance: 15_000,
+                        heading: 0,
+                        pitch: 70,
+                        roll: 0
+                    )
+                } catch {
+                    self.error = error
+                }
+            }
+            .errorAlert(presentingError: $error)
+    }
+    
+    /// Creates a dictionary renderer for styling with MIL-STD-2525D symbols.
+    /// - Returns: A new `DictionaryRenderer` object.
+    private func makeMIL2525DRenderer() async throws -> DictionaryRenderer {
+        // Creates a dictionary symbol style from a dictionary style portal item.
+        let portalItem = PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: .jointMilitarySymbologyDictionaryStyle
+        )
+        let dictionarySymbolStyle = DictionarySymbolStyle(portalItem: portalItem)
+        try await dictionarySymbolStyle.load()
+        
+        // Uses the "Ordered Anchor Points" for the symbol style draw rule.
+        let drawRuleConfiguration = dictionarySymbolStyle.configurations.first { $0.name == "model" }
+        drawRuleConfiguration?.value = "ORDERED ANCHOR POINTS"
+        
+        return DictionaryRenderer(dictionarySymbolStyle: dictionarySymbolStyle)
+    }
+    
+    /// Creates graphics from messages in an XML file.
+    /// - Returns: An array of new `Graphic` objects.
+    private func makeMessageGraphics() throws -> [Graphic] {
+        // Gets the data from the local XML file.
+        let messagesData = try Data(contentsOf: .mil2525dMessagesXMLFile)
+        let parser = MessageParser(data: messagesData)
+        
+        if parser.parse() {
+            // Creates graphics from the parsed messages.
+            return parser.messages.map { message in
+                let spatialReference = SpatialReference(wkid: .init(message.wkid!)!)
```

Comment:
I would advise against force unwrapping `message.wkid` because the optional value is data dependent.

You could use compactMap instead of map and return nil if a wkid cannot be created.